packages/iTwinUI-react/src/core/Tile/Tile.tsx

```diff
@@ -150,6 +161,8 @@ export const Tile = (props: TileProps) => {
           'iui-new': isNew,
           'iui-selected': isSelected,
           'iui-actionable': isActionable,
+          [`iui-${status}`]: !!status,
```

Comment:
I don't think we are adding disabled state to the tiles. Should we add that here?

Reply:
If we are adding disabled state to the tiles, we can add it as a prop separate from status. Similar to [L53](https://github.com/iTwin/iTwinUI-react/blob/main/packages/iTwinUI-react/src/core/Checkbox/Checkbox.tsx#L53) & [L71](https://github.com/iTwin/iTwinUI-react/blob/main/packages/iTwinUI-react/src/core/Checkbox/Checkbox.tsx#L71) in `Checkbox.tsx`

Reply:
We don't have disabled state examples for tile with buttons. When aria-disabled is added - main buttons and thumbnail buttons are still actionable.
Do we leave for our users to add disabled state to these buttons? Do we clone elements and add it ourselves?
@mayank99 @FlyersPh9 

![image](https://user-images.githubusercontent.com/83585998/198569086-16fe9e25-ea91-4c3e-9810-964d4d960e85.png)

Reply:
The user is in charge of specifying the button so I think they should disable it manually. Maybe we can provide an example and mention it in jsdoc.

Reply:
Disabled button looks a bit strange to me:
![image](https://user-images.githubusercontent.com/83585998/199024848-6ab06931-58e5-40fb-a239-b646ecebee5a.png)

Especially because thumbnails background is different kind of gray.
![image](https://user-images.githubusercontent.com/83585998/199026660-b7f6dfd3-a2bb-41a2-b917-ca1b16d912d9.png)

Reply:
yeah i'm not a fan of the disabled styling

Reply:
Leaving as is. It's css issue.